apps/costcontrol/elements/datausage-tab.html

```diff
@@ -16,20 +16,20 @@
 
         <ul class="toggle-data" data-type="list">
           <li id="mobileItem">
-            <gaia-checkbox id="mobileCheck">
-              <label>
-                <span data-l10n-id="mobile-usage">Mobile usage</span>
-                <span class="overview" id="mobileOverview"></span>
-              </label>
-            </gaia-checkbox>
+            <label class="pack-checkbox">
```

Comment:
Sorry it seems I've misinformed you, new gaia-checkbox web-component _does_ support RTL (now it just simple checkbox without any label inside), but it's still in a very early stage and nobody uses it in gaia + migrating Usage app to Bower (gaia-header, gaia-theme, gaia-icons should be switched to bower altogerher afaik) is a bit risky for v2.2 patch :)

I've filed a bug for that - https://bugzilla.mozilla.org/show_bug.cgi?id=1138432